src/explore-education-statistics-frontend/src/modules/data-catalogue/DataSetFilePage.tsx

```diff
@@ -214,6 +215,11 @@ export default function DataSetFilePage({ dataSetFileId }: Props) {
                       {orderBy(geographicLevels).join(', ')}
                     </SummaryListItem>
                   )}
+                  {lastUpdated && (
+                    <SummaryListItem term="Last updated">
+                      {new Date(lastUpdated).toDateString()}
```

Comment:
I think generally on the frontend, we use the `FormattedDate` in cases like this.